.travis.yml

```diff
@@ -0,0 +1,23 @@
+os: 
```

Comment:
Clippy etc are not tested in the CI scripts. Pls use safe-core `dev` branch travis and appveyor to base this and modifying the relevant parts (e.g. we don't need https://github.com/maidsafe/safe_core/blob/3a38924eb8a23a9c6ba82ad02834aaee2da30865/.travis.yml#L3 etc.). However we need all the rest of it (like rustfmt checks etc - basically all that safe-core does for both travis and appveyor)